hydra_base/lib/template/__init__.py

```diff
@@ -603,12 +632,14 @@ def delete_template(template_id, **kwargs):
     return 'OK'
 
 @required_perms("get_template")
-def get_templates(load_all=True, **kwargs):
+def get_templates(load_all=True, include_inactive=False, **kwargs):
     """
         Get all templates.
         Args:
             load_all Boolean: Returns just the template entry or the full
             template structure (template types and type attrs)
+            ignore_status Boolean: If true, returns all templates. If false, returns
```

Comment:
It's called `include_inactive` in the func def and this is the clearer name.

```suggestion
            include_inactive Boolean: If true, returns all templates. If false, returns
```

Reply:
done